source/slang/hlsl.meta.slang

```diff
@@ -7073,6 +7073,10 @@ ${{{{
 const char* diffDimensions[2] = {"x", "y"};
 for (auto xOrY : diffDimensions) {
 }}}}
+/// Take the partial derivative of `x` with respect to $(xOrY) in screen space.
+/// @param x The value to take partial derivative for.
+/// @return The partial derivative of `x`
+/// @remarks for SPIR-V, this function maps to `OpDPd$(xOrY)`.
```

Comment:
For SPIR-V

Reply:
'x' -> '$(xOrY)'

Reply:
I'm going to set up the gem workflow for previewing the rendered markup today so I can see what you mean, but I thought the parameter was named `x` in all cases, so line 7076, 7077, and 7078 where I wrote `x` I think it's correct, but I need to see the rendered result anyway.

Reply:
The rendered version looks like what I expected.  I think it's confusing to have the parameter named 'x' when dealing with x or y partial derivatives though.  The description of ddy is:
Take the partial derivative of `x` with respect to y in screen space.

Any objection to renaming the parameter p instead of x?

Reply:
I've updated the parameter name to p, as to not be confused with x,y